src/Module.zig

```diff
@@ -3408,7 +3428,8 @@ pub fn getErrorValue(mod: *Module, name: []const u8) !std.StringHashMapUnmanaged
     errdefer mod.global_error_set.removeAssertDiscard(name);
 
     gop.entry.key = try mod.gpa.dupe(u8, name);
-    gop.entry.value = @intCast(u16, mod.global_error_set.count() - 1);
+    gop.entry.value = @intCast(u16, mod.global_error_set.count());
```

Comment:
This is supposed to be tied to `inv_error_map` right? should be 

```suggestion
    gop.entry.value = @intCast(u16, mod.inv_error_map.items.len);
```

for clarity